linera-chain/src/manager.rs

```diff
@@ -589,7 +598,9 @@ impl From<&ChainManager> for ChainManagerInfo {
         ChainManagerInfo {
             ownership: manager.ownership.clone(),
             requested_proposed: None,
+            proposed_pending_blobs: BTreeMap::new(),
             requested_locked: None,
+            locked_pending_blobs: BTreeMap::new(),
```

Comment:
Would it be better to turn these into `Vec<Blob>`s? (In the `ChainManagerInfo`, _not_ in the `ChainManager`.) The recipient can't trust that the blob ID → blob association is correct anyway.